README.md

```diff
@@ -13,24 +13,24 @@ remote Opam repository:
 
 ## Layout of This Repository
 
-Packages are organised into namespaces. They are equivalent from the
-perspective of the [Opam] user.
+Packages are organised into namespaces:
 
 * `upstream`: packages that we don't control with fixed versions.
+* `upstream-extra`: packages that we don't control with fixed versions.
```

Comment:
I would add here that we don't need them as build dependencies. What do you think?